package.json

```diff
@@ -98,6 +98,8 @@
     "jsonwebtoken": "^8.3.0",
     "juice": "^5.2.0",
     "jwks-rsa": "^1.3.0",
+    "keymaster": "^1.6.2",
+    "linkify-it": "^2.1.0",
```

Comment:
Was this one added by mistake, or is it used somewhere?

Reply:
it came through in a merge I think? must be something that was added then removed from master? will remove